source/Ship.cpp

```diff
@@ -2662,7 +2662,10 @@ int Ship::RequiredCrew() const
 
 int Ship::CrewValue() const
 {
-	return max(Crew(), RequiredCrew()) + attributes.Get("crew equivalent");
+	int crewEquivalent = attributes.Get("crew equivalent");
+	if(attributes.Get("crew equivalent overrides"))
+		return crewEquivalent ? crewEquivalent : max(Crew(), RequiredCrew());
```

Comment:
I lied. *Now* it allows explicit zeros.
```suggestion
		return crewEquivalent;
```

Reply:
I should have spotted that ha
Even allows negative values